src/commonNative/kotlin/com/epam/drill/agent/State.kt

```diff
@@ -18,6 +19,21 @@ var isAsyncApp: Boolean
         _isAsyncApp.value = value.freeze()
     }
 
+private val _classScanDelay = AtomicReference(0L).freeze()
+
+var classScanDelay: Long
```

Comment:
Too much atomic variables. These age just configs and should be placed inside a config class.